app/models/course/assessment.rb

```diff
@@ -33,7 +33,7 @@ class Course::Assessment < ActiveRecord::Base
   #   maximum grade.
   #   @return [Fixnum]
   calculated :maximum_grade, (lambda do
-    Course::Assessment::Question.select { sum(course_assessment_questions.maximum_grade) }.
+    Course::Assessment::Question.unscoped.select { sum(course_assessment_questions.maximum_grade) }.
```

Comment:
The default scope makes the calculated query unable to perform, this is the downside of default scope